docs/EN_US/ContainerizedHPCC/ContainerizedMods/LocalDeployment.xml

```diff
@@ -31,44 +31,58 @@
 
     <para><programlisting>helm repo update</programlisting></para>
 
+    <para>You should update your local repo before any deployment to ensure
+    you have the latest code available.</para>
+
     <para>Expected response:</para>
 
-    <para><programlisting>Update Complete. Happy Helming!</programlisting></para>
+    <para><programlisting>Hang tight while we grab the latest from your chart repositories...
+...Successfully got an update from the "hpcc" chart repository
+Update Complete. Happy Helming!</programlisting></para>
   </sect1>
 
   <sect1 id="startdefault">
-    <title>Start a default system</title>
+    <title>Start a Default System</title>
 
     <para>The default helm chart starts a simple test system with Dali, ESP,
-    eclccserver, two eclagent queues (ROXIE and hThor mode), and one Thor
+    ECL CC Server, two ECL Agent queues (ROXIE and hThor mode), and one Thor
     queue.</para>
 
     <para><emphasis role="bold">To start this simple system:</emphasis></para>
 
-    <para><programlisting>helm install mycluster hpcc/hpcc --version=8.2.2 </programlisting></para>
+    <para><programlisting>helm install mycluster hpcc/hpcc --version=8.6.14</programlisting></para>
```

Comment:
you don't need the --version .. it will default to the latest.
This will be misleading if they've just done a helm repo update, and it a new version is available.. but then they're telling it not to use it by this saying use a specific [older] version.

Reply:
ignore this, I see explanation below, and that's fine.